vignettes/including-static-dashboards.Rmd

```diff
@@ -0,0 +1,103 @@
+
+You can host Flexdashboards on Crunch's Shiny infrastructure using Shiny's `includeHTML` function. What `includeHTML` does is add an HTML file to the shiny UI. If all you include in the shiny UI is the Flexdashboard html file, then the Shiny app effectively becomes a static site. 
+
+### Basic workflow
+Follow these steps to include a Flexdashboard in your Shiny app: 
+1) First create a new Rmarkdown document with a Flexdashboard output option. 
+1) Render the Rmd file into an html file
+1) Uploaded the rendered html file to your app's Jupyter folder
+1) In your shiny UI code add `includeHTML(<filename>)` to the `ui` object. Your `app.R` file might look something like this:
+
+```
+library(shiny)
+library(crunchy)
+# Render the dashboard
+ui <- fluidPage(
+    includeHTML("dashboard")
+)
+
+server <- function(input, output, session) {}
+
+# Run the application 
+shinyApp(ui = ui, server = server)
+```
+
+### Conditional dashboards
+
+Another great use of Flexdashboards is easily produce different dashboards for different users. To do this we'll make use of Shiny's `renderUI` function to conditionally create a different UI for different types of users.
+
+```
+ui <- fluidPage(
+    uiOutput("dashboard")
+)
+```
+
+This creates the ui object, but most of the work is created on the server side. We can then use the `shinyUser()` function to get a user record for the current viewer of the app. This record includes some useful pieces of information, including the user's name, their email address, and their Crunch preferences. You can use this object to conditionally display different dashboards for different users like so: 
+
+```
+server <- function(input, output, session) {
+    user <- getUser()
+    output$dashboard <- renderUI({
+        if (grepl("acme.com$", user()$email)) {
+            includeHTML("acme_dashboard.html")
+        } else {
+            includeHTML("globex_dashboard.html")
+        }
+    })
+    
+}
+```
+What this does is check the user's email address to identify clients from one company, and then displays a rendered dashboard for that company. If their email isn't a match, they get the Globex dashboard. You can see a full example app in the `/inst` directory. 
```

Comment:
The `inst/` directory isn't present when the package is installed, only in source. You could instead say `system.file("whatever-it-is-called.R", package="crunchy")`. And if you wanted, you could have that be a hyperlink to the `inst/` file on github.

Reply:
Perhaps also give an instruction of how to serve the app. You could even add a crunchy function to serve the demo apps.